pkgs/games/openrct2/default.nix

```diff
@@ -5,20 +5,20 @@
 
 let
   name = "openrct2-${version}";
-  version = "0.2.3";
+  version = "0.2.4";
 
   openrct2-src = fetchFromGitHub {
     owner = "OpenRCT2";
     repo = "OpenRCT2";
     rev = "v${version}";
-    sha256 = "01mj6jlbl2cn3wpk6sy34ldzdl0qykpn7fncznjykklj2nqzr4ig";
+    sha256 = "1rlw3w20llg36sj3bk50g661qw766ng8ma3p42sdkj8br9dw800h";
   };
 
   objects-src = fetchFromGitHub {
     owner = "OpenRCT2";
     repo = "objects";
-    rev = "v1.0.11";
-    sha256 = "1bh7mngpqnhzwnhhawq5y3a6hbvwxis2yagk4dcmc4w1fifq2y66";
+    rev = "v1.0.12";
+    sha256 = "0vfhyldc8nfvkg4d9kry669haxz2165walbxzgza7pqpnd7aqgrf";
   };
 
```

Comment:
The 0.2.4-release introduced a [replay](https://github.com/OpenRCT2/replays) module in its CMakeLists.txt. This sub-project should be added like the objects and title sequences.

Reply:
`replays` is only used for tests, which we don't currently run. 

If we want to run tests then we'll need to include [googletest](https://github.com/google/googletest/) in the derivation. I'm not sure if there's a quick and easy way to disable the download, so we might need to patch `test/tests/CMakeLists.txt` as well.